src/SprykerEco/Zed/Optivo/Business/Api/Adapter/Http/Guzzle.php

```diff
@@ -1,10 +1,16 @@
 <?php
 
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
 namespace SprykerEco\Zed\Optivo\Business\Api\Adapter\Http;
 
 use GuzzleHttp\Client;
 use GuzzleHttp\Exception\RequestException;
 use GuzzleHttp\Psr7\Request;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
 use SprykerEco\Zed\Optivo\Business\Exception\ApiHttpRequestException;
 use SprykerEco\Zed\Optivo\OptivoConfig;
 
```

Comment:
I think it's better to rename this class to GuzzleAdapter.

Reply:
done